docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,119 @@
+
+## Billing
+
+### What does a billing cycle start for a plan?
+
+For the Free plan, the billing cycle starts on the first day of the calendar month.
+
+For all other paid plans, the billing cycle starts from the subscription date of that plan.
+
+### How do I update my billing information?
+
+To update your Organization account's billing information, make sure that you have either [**Owner** or **Admin** role](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role. After verifying the role:
+
+- Go to your [**Account or Organization settings** > **Billing**](https://expo.dev/settings/billing), and click on **Update billing information**.
+- On Stripe's Portal, under **Billing information**, click on **Update information** to update billing-related information such as billing name, email, address, and tax ID Billing information.
+
+See [Manage billing information](/billing/manage/#manage-billing-information) for more details.
+
+### Can I update the billing info for my last invoice?
+
+No. Updating billing information will only be reflected on the next invoice.
+
+### Can you email me the receipts?
+
+No. Your account's Owner or Admin can download them. See [Download a receipt](/billing/invoices-and-receipts/#download-and-view-a-receipt) for more information.
+
+### How can I reduce the amount of EAS Build usage by using EAS Update?
+
+Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. For native code changes, you can also automate the build process using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime).
+
+For more information, see [How to optimize build usage](/billing/usage-based-pricing/#how-to-optimize-build-usage).
+
+### How do I estimate my next on-demand bill?
+
+To estimate your next on-demand bill, go to [**Billing**](https://expo.dev/settings/billing). Under **Usage this month**, you will find a summary of your EAS Build usage based on the [resource class](/build/eas-json/#selecting-resource-class), EAS Update usage based on monthly updated users and global edge bandwidth, and the amount spent for both.
+
+See [How usage-based pricing works](/billing/usage-based-pricing/#how-usage-based-pricing-works) for more information.
+
+### What is a monthly updated user?
+
+A monthly updated user counts only once per billing cycle, regardless of how many updates this user downloads. See [How are "monthly updated users" counted for a billing cycle](/eas-update/faq/#how-are-monthly-updated-users-counted-for-a-billing-cycle) for more information.
```

Comment:
Let's add one sentence to explain what it is:

> A monthly updated user is a unique user who downloads an update via EAS Update in a single month.

Also, we need to qualify the first sentence:

> regardless of how many updates this user downloads **in a given billing month/period**.